fighthealthinsurance/models.py

```
@@ -2841,3 +2902,118 @@ class UsedDeleteToken(models.Model):
+                fields=[
+                    "procedure_code",
+                    "geographic_area",
+                    "effective_date",
+                    "expires_date",
+                ],
+                name="ucr_rate_lookup_idx",
+            ),
+            models.Index(
+                fields=["source", "effective_date"], name="ucr_rate_source_idx"
+            ),
+        ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)